### PR TITLE
Revert "Add Terraform Support For Cloud SQL Preferred Zone Selection Feature"

### PR DESCRIPTION
**Pull request by trodge**
Reverts GoogleCloudPlatform/magic-modules#9341

Unintended update to dependencies, reverting for now

### CODE REVIEW

**Review by rileykarson — APPROVED** [review]

**Comment by shuyama1** [comment]
We'll probably need to wait until the VCR tests finish to merge the revert PR, as we need to get the cassettes corrected unless no tests are affected in the initial PR. I'll copy the cassettes from this PR branch to the main if there's any new one recorded after tests are done. Should be good.

**Pull request merged**
